server/database/queries/orderMeal.js

```diff
@@ -0,0 +1,11 @@
+const connection = require('../config/connection');
```

Comment:
Add the two queries in one file and call it `order`

i.e.
- queries/
    - order --> **contains** `insertOrder` and `insertOrderMeals`